packit_config_checker.py

```diff
@@ -484,6 +550,11 @@ def show_diff(old_package_config: str, new_package_config: str):
     GITHUB_TOKEN env var (and additionally
     GITLAB_TOKEN, GITLAB_GNOME_TOKEN, GITLAB_FREEDESKTOP_TOKEN depending on the affected projects).
 
+
+    For working with src.fedoraproject.org (as packit user), you need to:
+        - run fkinit -u packit # take credentials from bitwarden
```

Comment:
Since we are taking SSH key from `secrets`, we may as well use the keytab from there as well:

```suggestion
        - run kinit packit@FEDORAPROJECT.ORG -k -t /path/to/deployment/secrets/packit/prod/fedora.keytab
```